package.json

```diff
@@ -30,7 +30,9 @@
     "url": "https://github.com/blackbaud/help-client/issues"
   },
   "homepage": "https://github.com/blackbaud/help-client#readme",
-  "dependencies": {},
+  "dependencies": {
+    "@types/tapable": "^0.2.4"
```

Comment:
Thoughts on making this a "hard" dependency? (Relative dependencies are better for `peerDependencies`.)

Reply:
If it still solves my issue with the build, I am all for that.